cmd/capture.go

```diff
@@ -49,7 +49,7 @@ by passing the function name symbol and the binary args.
 		for _, functionSymbol := range functionSymbolList {
 			syscalls, err := captor.Capture(functionSymbol, args, captureOpts)
 			if err != nil {
-				return fmt.Errorf("error capturing syscall: %w", err)
+				fmt.Printf("error capturing syscall: %w", err)
```

Comment:
There is no %w with Printf if I'm not wrong

```suggestion
				fmt.Printf("error capturing syscall: %v", err)
```

https://pkg.go.dev/fmt#hdr-Printing